cmake/Modules/G4OptionalComponents.cmake

```diff
@@ -177,7 +177,7 @@ option(GEANT4_USE_SYSTEM_PTL "Use system zlib library" OFF)
 if(GEANT4_USE_SYSTEM_PTL)
   find_package(PTL REQUIRED)
   # Backward compatibility for sources.cmake using the variable
-  set(PTL_LIBRARIES PTL::PTL)
+  set(PTL_LIBRARIES PTL::ptl-shared)
```

Comment:
This should be: 

```cmake
  set(PTL_LIBRARIES PTL::ptl)
```

See: 
https://github.com/jrmadsen/PTL/blob/317caa7e0c3d04952908ea41e9db05ca2e96c6dc/cmake/Templates/PTLConfig.cmake.in#L142

Reply:
Note: in the link above, this is PTL v1.0.1 which changed some stuff w.r.t. PTLConfig.cmake but the code w.r.t. `PTL::ptl` did not change so it will work fine with PTL v1.0.0

Reply:
Hi @jrmadsen ,
Update --  done.